src/components/SignUpForm/styles.scss

```
@@ -0,0 +1,37 @@
+@import "../../styles/main.scss";
```

Actually, it has to be with single code. 
```suggestion
@import '../../styles/main.scss';
```